css/index.css

```diff
@@ -112,6 +128,46 @@ p {
     justify-content: center;
 }
 
```

Comment:
You are making good progress with the index.html.  Check on styling the nav components so they look like the design file and start working on getting flexbox applied to the page.